hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmBucketArgs.java

```diff
@@ -187,6 +187,27 @@ public Map<String, String> toAuditMap() {
     if (this.ownerName != null) {
       auditMap.put(OzoneConsts.OWNER, this.ownerName);
     }
+    if (this.quotaInBytesSet && quotaInBytes > 0 ||
+        !(this.quotaInBytes == OzoneConsts.QUOTA_RESET)) {
+      auditMap.put(OzoneConsts.QUOTA_IN_BYTES,
+          String.valueOf(this.quotaInBytes));
+    }
+    if (this.quotaInNamespaceSet && quotaInNamespace > 0 ||
+        !(this.quotaInNamespace == OzoneConsts.QUOTA_RESET)) {
+      auditMap.put(OzoneConsts.QUOTA_IN_NAMESPACE,
+          String.valueOf(this.quotaInNamespace));
+    }
+    if (this.bekInfo != null) {
+      auditMap.put(OzoneConsts.BUCKET_ENCRYPTION_KEY_NAME,
+          this.bekInfo.getKeyName());
+    }
+    if (this.defaultReplicationConfig != null) {
+      auditMap.put(OzoneConsts.REPLICATION_TYPE, String.valueOf(
+          this.defaultReplicationConfig.getType()));
+      auditMap.put(OzoneConsts.REPLICATION_FACTOR,
```

Comment:
"Factor" does not apply to EC, so I suggest using `REPLICATION_CONFIG` instead.

```suggestion
      auditMap.put(OzoneConsts.REPLICATION_CONFIG,
```